src/GrassPokemon.java

```diff
@@ -0,0 +1,50 @@
+public class GrassPokemon extends Pokemon {
+    private int grassPower;
+    private String grassAttack;
+
+    public GrassPokemon(String name, int level, int grassPower, String grassAttack) {
+        super(name, level);
+        this.grassPower = grassPower;
+        this.grassAttack = grassAttack;
+    }
+
+    public void useGrassAttack() {
+        System.out.println(getName() + " used " + grassAttack + "!");
```

Comment:
Je gebruikt hier "getName()" en dat werkt prima, maar het is duidelijker om "super.getName()" te gebruiken. Dan zie je in 1 oogopslag dat deze variabele uit de super overerft wordt.